index.js

```diff
@@ -26,43 +28,120 @@ function sortToArray(map) {
             Array.prototype.push.apply(list, pa);
     }
 
+
     return list;
 }
 
-module.exports = (package, size) => {
+function similarEnoughToSkip(inputPkgDetails, candidatePkgDetails) {
```

Comment:
Eliminates packages which come from the same origin/homepage or just look like forks.

Its a very crud heuristic and can definitely be improved.